build.sbt

```diff
@@ -11,6 +11,14 @@ inThisBuild(
   )
 )
 
+scalacOptions ++= {
+  if (scalaVersion.value == Scala211) {
+    Seq.empty
+  } else {
+    Seq("-target:jvm-1.6")
+  }
+}
```

Comment:
I can't figure out what this is for.

Reply:
This is mistake as it is. I wanted to fix compilation problem in 2.11.
```
[error] /home/runner/work/zio-nio/zio-nio/nio/src/main/scala/zio/nio/file/Files.scala:138:99: Static methods in interface require -target:jvm-1.8
[error]       effectBlocking(JFiles.walk(path.javaPath).sorted(Comparator.reverseOrder[java.nio.file.Path]()))
[error]                                                                                                   ^
[error] one error found
```
And for 2.11 I need to set taget:jvm-1.8. I changed it now